modules/oadp-configuring-aws-md5sum.adoc

```diff
@@ -0,0 +1,75 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.adoc
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-configuring-aws-md5sum_{context}"]
+= Configuring the backup storage location with MD5 checksum algorithm
+
+You can configure the Backup Storage Location (BSL) in the Data Protection Application (DPA) to use a MD5 checksum algorithm for {aws-short} S3 and S3 compatible storage providers. The checksum algorithm calculates the checksum for uploading and downloading objects to S3. You can use one of the following options to set the `checksumAlgorithm` field in the `spec.backupLocations.velero.config.checksumAlgorithm` section of the DPA.
+
+* `CRC32`
+* `CRC32C`
+* `SHA1`
+* `SHA256`
+
+[NOTE]
+====
+You can also set the `checksumAlgorithm` field to an empty value to skip the MD5 checksum check. 
+
+If you do not set a value for the `checksumAlgorithm` field, then the default value is set to `CRC32`.
+====
+
+.Prerequisites
+
+* You have installed the {oadp-short} Operator.
+* You have configured {aws-short} S3, or S3 compatible object storage as a backup location.
```

Comment:
Here as well ensure that we're referring to the correct thing.

Reply:
Done. thanks.